lti_consumer/lti_1p3/consumer.py

```diff
@@ -193,25 +195,27 @@ def set_resource_link_claim(
 
     def set_launch_presentation_claim(
             self,
-            document_target="iframe"
+            document_target="iframe",
+            return_url=None,
     ):
         """
         Optional: Set launch presentation claims
 
         http://www.imsglobal.org/spec/lti/v1p3/#launch-presentation-claim
         """
-        if document_target not in ['iframe', 'frame', 'window']:
+        if document_target and document_target not in ['iframe', 'frame', 'window']:
```

Comment:
this means it's ok to have document_target = None or ''?

Reply:
I think the first condition (`if document_target`) should check that document_target is not None right?

Reply:
Oh wait I think I understand what you're saying @ashultz0. I guess, should there be a check for `if not document_target`?